### ISSUE TITLE
[BUG] [Linter] False positive on H037

### ISSUE THREAD

**Issue by malo-cayocca** [Riverside-Healthcare/djLint#679]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: ubuntu 20.04
 - Python Version 3.8.10
 - djLint Version 1.30.0
 - template language: django templates

## Issue
I get a false positive of H037 Duplicate attribute found on a `rect` tag with both `x` and `rx` attributes.

## How To Reproduce
Here is a minimal reproduction : 
```
<rect x="2" y="3" rx="1" />
```

## Contents of .djlintrc/pyproject.toml [tool.djlint]
```
[tool.djlint]
require_pragma = true
profile = "django"
custom_blocks = "..."
custom_html = "..."
blank_line_after_tag = "extends, endblock"
max_line_length = 120
ignore = "H021,H023,H030,H031"
```

**Comment by christopherpickering** [comment]
Thanks for catching... beginners mistake 😄 I will fix in the next release.
